enhancements/update/distribute-secondary-metadata-as-container-image.md

```diff
@@ -0,0 +1,185 @@
+---
+title: cincinnati-graph-data-package-distribute-and-consume-in-a-container-image-repository
+authors:
+  - "@steveeJ"
+reviewers:
+  - @openshift/openshift-team-cincinnati-maintainers
+  - @openshift/openshift-team-cincinnati
+  - @openshift/team-cincinnati-operator
+approvers:
+  - @crawford
+  - @LalatenduMohanty
+  - @sdodson
+creation-date: 2020-05-07
+last-updated: 2020-05-07
+status: provision
+see-also:
+-
+replaces: []
+superseded-by: []
+---
+
+# cincinnati-graph-data: package, distribute, and consume in a container image repository
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [x] Graduation criteria
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions
```

Comment:
I would move the Open Questions section towards the end or at least somewhere after the enhancement itself has been described in detail. The questions will be easier to understand and assess with an understanding of the enhancement itself. Especially for someone who has not been involved in the many internal discussions had by the OTA team.

Reply:
I see your point. I kept the template's order for the sections, and I do see an advantage of this order as well. It makes sense to resolve the open questions before reviewing the full enhancement, as that may change significantly after these questions have been resolved.

Reply:
So the intent of Open Questions section is to get the questions answered before the enhancement is reviewed externally - that is, by those outside of OTA team - and then remove this section and update the enhancement per the question resolutions?

Reply:
That's almost how I understand it. My interpretation differs insofar that it's not limited it to those outside of OTA, but to anyone who can contribute to resolving it.
I could also see us leaving some open questions there if we decide they're not in scope.

Reply:
> I could also see us leaving some open questions there if we decide they're not in scope.

If we decide it's out of scope, we should have a *Non-Goals* paragraph about it.  *Open Questions*, as described in the template, is just for:

>  ...areas of the design that require closure before deciding to implement the design...

Reply:
Thanks @wking, that makes sense. Still a few questions to resolve there!